chsdi/models/vector/uvek.py

```diff
@@ -14,20 +14,52 @@
 
 
 class OevHaltestellen (Base, Vector):
```

Comment:
please factorize, have a look at:
https://github.com/geoadmin/mf-chsdi3/blob/master/chsdi/models/vector/stopo.py#L175
for instance